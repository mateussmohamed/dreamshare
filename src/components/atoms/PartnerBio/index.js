import styled from 'styled-components'

const PartnerBio = styled.p`
  ${props => props.theme.typography.robotoStyle};
  font-size: 14px;
  text-align: center;
  margin: 0 0 15px 0;
`

export default PartnerBio
