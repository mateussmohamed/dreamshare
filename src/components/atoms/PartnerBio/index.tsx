import styled from 'styled-components'

export const PartnerBio = styled.p`
  ${props => props.theme.typography.robotoStyle};
  font-size: 14px;
  text-align: center;
  margin: 0 0 15px 0;
`
