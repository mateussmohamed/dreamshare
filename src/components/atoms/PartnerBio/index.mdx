---
name: PartnerBio
menu: Atoms
---

import { Playground, PropsTable } from 'docz'
import PartnerBio from './index'

# PartnerBio
<PropsTable of={PartnerBio} />

<Playground>
  <PartnerBio />
</Playground>
