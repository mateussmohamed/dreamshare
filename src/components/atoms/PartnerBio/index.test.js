import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import PartnerBio from './index'

describe('[<PartnerBio />]', () => {
  const wrap = (props = {}) => shallow(<PartnerBio {...props} />)

  test('matches to snapshot', () => {
    const wrapper = wrap({})

    expect(wrapper).toMatchSnapshot()
  })
})
