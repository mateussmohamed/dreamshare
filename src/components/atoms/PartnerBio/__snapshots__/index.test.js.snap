// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<PartnerBio />] matches to snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "PartnerBio-sc-1b2g24y-0",
        "isStatic": false,
        "rules": Array [
          [Function],
          ";font-size:14px;text-align:center;margin:0 0 15px 0;",
        ],
      },
      "displayName": "PartnerBio",
      "render": [Function],
      "styledComponentId": "PartnerBio-sc-1b2g24y-0",
      "target": "p",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
